redaction/draft-strad-trans-redaction.md

```diff
@@ -88,6 +94,64 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC2119].
 
+# Wildcard Certificates
+
+A certificate containing a DNS-ID [RFC6125] of `*.example.com` could be used to
+secure the domain `topsecret.example.com`, without revealing the string
```

Comment:
Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/10cd9abd14675c02d71ff0d0e399b143d7f0dd73

(FWIW, I think either "string" or "label" would be fine here.  The string is indeed a label, but if it had no significance beyond being just a label there wouldn't be any need to keep it secret!)